10/src/javascripts/wd.js

```diff
@@ -0,0 +1,74 @@
+module.exports = function () {
+    let apps = [];
+
+    function renderComponents(app) {
+        app.components.forEach(component => {
+            let tags = document.getElementsByTagName(component.name);
```

Comment:
I would recommend to use `querySelectorAll(cssSelector)` method of the `document`. If your implementation of looking for element would change, you would need to change the method. For example, if you change fot `id`, you would use `getElementById()`, if for `class` you would use `getElementsByClassName()`. But you can use only one method for all cases.